Bruin Bite/Network/MenuAPI.swift

```diff
@@ -278,10 +283,10 @@ extension API{
                     completion(menus)
                 }
                 catch{
-                    print("error parsing")
+                    Logger.shared.handle(type: .error, message: "Could not parse detailed menu")
                 }
             case let .failure(error):
-                print(error)
+                Logger.shared.handle(type: .error, message: "getDetailedMenu: \(error)")
```

Comment:
Same as previous. Change error message to "getDetailedMenu failed with error: \(error)"